dds_web/static/swaggerv3.yaml

```diff
@@ -0,0 +1,2123 @@
+        the actual file to the system you need to get the s3 keys in the /s3/proj endpoint
+      operationId: newFile
+      parameters:
+        - $ref: "#/components/parameters/defaultHeader"
+        - $ref: "#/components/parameters/projectID"
+      responses:
+        "401":
+          $ref: "#/components/responses/UnauthorizedToken"
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "500":
+          $ref: "#/components/responses/InternalServerlError"
+        "200":
+          description: succesful operation
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: "File data_seq.txt added to db."
+      requestBody:
+        $ref: "#/components/requestBodies/Files"
+    put:
+      tags:
+        - files
+      summary: Update an existing file
+      description: Updates the metadata of a file in the DB. The name will be used
+        to find the file. The rest of the fields will be updated.
+      operationId: updateFile
+      parameters:
+        - $ref: "#/components/parameters/defaultHeader"
+        - $ref: "#/components/parameters/projectID"
+      responses:
+        "401":
+          $ref: "#/components/responses/UnauthorizedToken"
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "500":
+          $ref: "#/components/responses/InternalServerlError"
+        "200":
+          description: succesful operation
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: "File data_seq.txt updated in the db."
+      requestBody:
+        $ref: "#/components/requestBodies/Files"
+  /file/match:
+    get:
+      tags:
+        - files
+      summary: Get name in bucket for all files specified
+      operationId: matchFiles
+      parameters:
+        - $ref: "#/components/parameters/defaultHeader"
+        - $ref: "#/components/parameters/projectID"
+        - $ref: "#/components/parameters/filesToQuery"
+      responses:
+        "401":
+          $ref: "#/components/responses/UnauthorizedToken"
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "500":
+          $ref: "#/components/responses/InternalServerlError"
+        "200":
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  files:
+                    type: array
+                    items:
+                      type: string
+                    example: [name_in_bucket, name_in_bucket_2]
+  /files/list:
+    get:
+      tags:
+        - files
+      summary: Get a list of files and folders within an specific folder
```

Comment:
This summary should be written in swagger.yalm exactly the same as here. 
This applies for all the endpoint methods where summaries/descriptions/etc differ